docs/pages/app-signing/security.mdx

```diff
@@ -39,6 +39,22 @@ You will not be able to update your app on Google Play. You may want to download
 
 Expo tools never ask you to provide your Google account credentials.
 
+## Android submit credentials
+
+### Google Service Account Key
+
+Google Service Account Key is the authentication method required to be used to submit an Android app to the Google Play Store using the EAS Submit service. This key is permanently stored on the Expo servers (unless deleted by a developer) while being encrypted using [KMS](https://cloud.google.com/security/products/security-key-management) when at rest.
```

Comment:
```suggestion
Google Service Account Key is the authentication method used to submit an Android app to the Google Play Store with EAS Submit. This key is stored on Expo servers and encrypted using [KMS](https://cloud.google.com/security/products/security-key-management) when at rest. The key will remain on Expo servers to be re-used for subsequent submissions, and it can be removed at any time by a user with the requisite permissions.
```